src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -0,0 +1,93 @@
+package api.loans;
+
+import static api.support.matchers.LoanMatchers.hasLoanProperty;
+import static api.support.matchers.LoanMatchers.hasOpenStatus;
+import static api.support.matchers.LoanMatchers.loanItemIsDeclaredLost;
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import api.support.APITests;
+import api.support.http.InterfaceUrls;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.folio.circulation.support.http.client.Response;
+import org.folio.circulation.support.http.client.ResponseHandler;
+import org.junit.Test;
+
+public class DeclareLostAPITests extends APITests {
```

Comment:
What happens if an attempt is made to declare an item lost for a loan that does not exist?

What happens if an attempt is made to declare an item lost for a loan that is closed?

I would prefer that the behaviour in situations like this are described with tests.

Reply:
Clarificatins from Emma about invoking this on closed loans:
`That shouldn't be allowed. Only an open loan can have an item that is declared lost. `

We'll return an error message in this case.